### ISSUE TITLE
Should it fail sbt when it fails to parse .travis.yml? 

### ISSUE THREAD

**Issue by kailuowang** [dwijnand/sbt-travisci#33]
Namely here. 
https://github.com/dwijnand/sbt-travisci/blob/master/src/main/scala/sbttravisci/TravisCiPlugin.scala#L79-L84
The rationale for failing outright is that such a failure should fail the CI build.

**Comment by dwijnand** [comment]
That behaviour comes from #8, so we'd need to evaluate changing it with @djspiewak's use-case.  I'd be happy to have an opt-in "strict mode".

**Comment by djspiewak** [comment]
Yeah it's really for when you don't have a .travis.yml. I think failure to parse is probably something that could be errored on, but to what end? Also happy for an opt-in strict mode (or if there's enough demand, an opt-out).
